JS/index.js

```diff
@@ -34,4 +34,4 @@ const ClassificationLevel = require('./models/ClassificationLevel');
     console.log(`Military airport sorted by max distance: ${Airport.print(militaryAirport.sortByMaxDistance())}`);
     console.log(`Passenger airport sorted by max speed: ${Airport.print(passengerAirport.sortByMaxSpeed())}`);
     console.log(`Plane with max passenger capacity: ${Airport.print(passengerAirport.getPassengerPlaneWithMaxPassengersCapacity())}`);
-})();
```

Comment:
Why did you delete parentheses here?
This is a special syntax for modules in JavaScript.

Reply:
Fixed. Cleared out index.js.